spec/overview.html

```diff
@@ -73,6 +73,10 @@ <h1>Implementation Dependencies</h1>
       </li>
     </ul>
 
+    <p>
+      Throughout this specification, behaviour that is dependent on implementation and locale is referred to as ILD. Behaviour that is dependent on implementation, locale, and numbering system is referred to as ILND.
```

Comment:
These initialisms should be auto-linked, and ideally also continue to convey their source phrasing.
```suggestion
      Throughout this specification, implementation- and locale-dependent behaviour is referred to as <dfn>ILD</dfn>, and implementation-, locale-, and numbering system-dependent behaviour is referred to as <dfn>ILND</dfn>.
```